Core/Contributions/JGFoster/StoreString for Views.pax

```diff
@@ -1293,14 +1280,14 @@ storeForViewOn: aStream
 		space;
 		display: #fromMilliseconds:;
 		space.
-	milliseconds storeForViewOn: aStream.
+	self millisecond storeForViewOn: aStream.
```

Comment:
Time>>millisecond returns the number of milliseconds in the fractional part of the Time’s total number of seconds. To get a precise round trip use asSeconds and fromSeconds:

Reply:
@jgfoster, I’ll go ahead and merge this code as is if you would prefer to fix it later. I’d guess that having the correct Time instance in a view resource is unlikely to matter. 
I’ll give it a couple more hours then press go, or let me know otherwise.

Reply:
Either way is fine (no one is complaining about the methods that don't compile). I'll try to get to it today.